app/views/candidate_interface/continuous_applications/course_choices/review_interruption/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :title, t('page_titles.application_review_and_submit', provider_name: @application_choice.provider.name) %>
+<% content_for(:before_content, govuk_back_link_to(candidate_interface_continuous_applications_choices_path)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <div class="app-grid-column--grey">
+      <h1 class="govuk-heading-l">Give your application the best chance of success</h1>
+      <p class="govuk-body">Your personal statement is <%= "#{@word_count} #{'word'.pluralize(@word_count)}" %>.</p>
+      <p class="govuk-body">90% of successful candidates write about 500 words or more for their personal statement.</p>
```

Comment:
Would we consider putting the `RECOMMENDED_PERSONAL_STATEMENT_WORD_COUNT` as a constant on the `ApplicationForm` model, so we can use the value here?

Reply:
Done